tests/fugue/utils/test_interfaceless.py

```diff
@@ -104,12 +104,13 @@ def test_parse_function():
     raises(TypeError, lambda: _parse_function(f15, "^e?(c|[dl]+)x*$", "n"))
     _parse_function(f14, "^0?e?(c|[dl]+)x*$", "n")
     _parse_function(f16, "^0e?(c|[dl]+)x*$", "n")
+    _parse_function(f33, "^$", "q")
 
 
 def test_function_wrapper():
-    for f in [f20, f21, f212, f22, f23, f24, f25, f26, f30, f31, f32]:
+    for f in [f20, f21, f212, f22, f23, f24, f25, f26, f30, f31, f32, f34]:
         df = ArrayDataFrame([[0]], "a:int")
-        w = FunctionWrapper(f, "^[ldsp][ldsp]$", "[ldsp]")
+        w = FunctionWrapper(f, "^[ldspq][ldspq]$", "[ldspq]")
```

Comment:
Places using FunctionWrapper need to update. You figured out how to update this test, but extension conversion logics need to be updated. And tests should be added